lib/pw_dict.go

```diff
@@ -62,12 +67,19 @@ func Pw_checkweak_load() (err error) {
 		Dbgf("Loading Weak Password Dictionary %q... done (%d passwords)", f, cntpw)
 	}
 
-	Logf("Loaded %d Weak Password Dictionaries with %d unique passwords", cnt, len(pw_dict))
+	Logf("Loaded %d Weak Password Dictionaries with %d unique passwords", pw_dicts, len(pw_dict))
 
 	return
 }
 
+// Pw_checkweak checks if a password is in our weak password list
 func Pw_checkweak(password string) (isweak bool) {
 	_, isweak = pw_dict[strings.ToLower(password)]
 	return
 }
+
+// Pw_details provides details about te password dictionary checker (used by system_report)
+func Pw_details() (msg string) {
+	fmt.Sprintf("Password Dictionary Checker: Loaded %d Weak Password Dictionaries with %d unique passwords", pw_dicts, len(pw_dict))
```

Comment:
Did a : git cherry-pick 2ba715eb6732282840e0d372e334af9880bd815c

and it is now in this branch too. Closing the #95 MR though, as otherwise it would be a dupe.